src/components/icons/baseIcon/baseIcon.tsx

```diff
@@ -0,0 +1,29 @@
+import { FC, MouseEventHandler, ReactElement, ReactNode } from 'react';
+import classNames from 'classnames/bind';
+import styles from './baseIcon.module.scss';
+
+const cx = classNames.bind(styles);
+
+export interface BaseIconProps {
+  children: ReactNode;
+  className?: string;
+  disabled?: boolean;
+  onClick?: MouseEventHandler<HTMLSpanElement>;
+}
+
+// DS link - https://www.figma.com/file/gjYQPbeyf4YsH3wZiVKoaj/%F0%9F%9B%A0-RP-DS-6?type=design&node-id=136-2&mode=design&t=Ri0jf5ugCvGn3z3b-0
+export const BaseIcon: FC<BaseIconProps> = ({
+  children,
+  className,
+  disabled = false,
+  onClick,
+}): ReactElement => {
```

Comment:
if it's not a part of code style, we can skip using FC because it's using by default 
```suggestion
export const BaseIcon = ({
  children,
  className,
  disabled = false,
  onClick,
}:BaseIconProps
): ReactElement => {
```

Reply:
It's actually more of a code style thing.
Also, I get helpful type tips from the IDE for optional props.

![image](https://github.com/reportportal/ui-kit/assets/31861805/f63c828e-f741-4052-bff8-91358c642d03)

We can discuss it additionally if needed.